packages/cli/examples/helpers.ts

```diff
@@ -1,101 +1,100 @@
 /* eslint-disable @typescript-eslint/camelcase */
-import {
-  logs
-} from "@cosmwasm/sdk";
+import { FeeTable } from "@cosmwasm/sdk/types/signingcosmwasmclient";
```

Comment:
Exported properly now (https://github.com/CosmWasm/cosmwasm-js/pull/151)

Reply:
Thanks. I will rebase and update